apache2/acmp.c

```diff
@@ -251,6 +251,7 @@ static void acmp_add_node_to_parent(acmp_node_t *parent, acmp_node_t *child) {
     }
 }
 
+#if 0
```

Comment:
Nice catch. It seems like we don't use this function at all - do we want to keep this unused code?

Reply:
Removed in 9d9a72734988bbd4f101cbea0d5df60eabcc1435